docs/yaml/objects/feature.yaml

```diff
@@ -85,3 +85,63 @@ methods:
       type: str
       default: "''"
       description: The error Message to print if the check fails
+
+- name: enable_if
+  returns: feature
+  since: 1.1.0
+  description: |
+    Returns the object itself if the value is false; an error if the object is
+    `'disabled'` and the value is true; an enabled feature if the object
+    is `'auto'` or `'enabled'` and the value is true.
+
+  example: |
+    `enable_if` is useful to restrict the applicability of `'auto'` features,
+    particularly when passing them to [[dep]]:
```

Comment:
Passing them to `[[dependency]]` as in the function, not `[[dep]]` as in the returned object.